ui/src/main/java/edu/wpi/grip/ui/pipeline/input/InputSocketController.java

```diff
@@ -64,7 +64,9 @@
     @FXML
     public void initialize() {
         this.identifier.setText(this.socket.getSocketHint().getIdentifier());
-        this.type.setText(this.socket.getSocketHint().getType().getSimpleName());
+        if (!(this.socket.getSocketHint().getType().getSimpleName().contains("Enum"))) {
+            this.type.setText(this.socket.getSocketHint().getType().getSimpleName());
+        }
         this.handle = socketHandleViewFactory.create(this.socket);
         root.add(this.handle, 0, 0);
```

Comment:
Apparently this makes GRIP angry as it needs the handle for the XML. I agree that this might be something to fix later though.